v2g-liberty/rootfs/root/appdaemon/apps/v2g-liberty/modbus_evse_client.py

```diff
@@ -1115,11 +1034,8 @@ async def __force_get_register(self, register: int, min: int, max: int) -> int:
 
             if result is not None:
                 try:
-                    result = int(float(result.registers[0]))
-                    if result > self.HALF_MAX_USI:
-                        # This represents a negative value.
-                        result = result - self.MAX_USI
-                    if min <= result <= max:
+                    result = self.__get_2comp(result.registers[0])
```

Comment:
This looks like an unrelated change. Did you mean to include this?

Reply:
You are right, unrelated but relevant. Will take more care is splitting up in the future.